### PR TITLE
Allow users to specify their own json parser.

### PR DESCRIPTION
**Pull request by euresti**
This will permit things like using other json parsers or passing
arguments (e.g. `parse_float=Decimal`) to the parser method.

### CODE REVIEW

**Comment by euresti** [comment]
@pbernasconi any comments?  Thanks!

File: plaid/client.py [hunk 1]
```diff
@@ -39,7 +40,8 @@ def __init__(self,
                  suppress_warnings=False,
                  timeout=DEFAULT_TIMEOUT,
                  api_version=None,
-                 client_app=None):
+                 client_app=None,
+                 json_parser=json.loads):
```
Comment (pbernasconi) [inline plaid/client.py]: What do you think about putting this under `_options`? We follow this pattern for other endpoints, and could allow for more flexibility in the future for further overrides (eg: https://github.com/plaid/plaid-python/pull/182)

**Comment by pbernasconi** [comment]
Added a comment, but supportive of this! also currently working on the  CI tests, they've been failing for the last week and will hopefully have them fixed in time to merge this change

File: plaid/client.py (hunk 1 above)
Comment (euresti) [reply plaid/client.py]: @pbernasconi ping?

**Comment by cgfarmer4** [comment]
Closing due to age. Please let me know if this is still an issue.

**Comment by euresti** [comment]
Well it is an issue, we just workaround it by monkeypatching.

```
from plaid.internal import requester

class _DecimalJson:
    """
    Minimal version of json that loads floats as Decimal.

    This is used to monkey patch the json module used in plaid.
    If https://github.com/plaid/plaid-python/issues/136 is ever fixed we can remove
    this.
    """

    @staticmethod
    def loads(s: Union[str, bytes, bytearray], **kw: Any) -> Any:
        return json.loads(s, parse_float=Decimal, **kw)


requester.json = _DecimalJson
```

**Pull request closed without merging**